packages/dds/tree/src/simple-tree/leafNodeSchema.ts

```diff
@@ -32,7 +33,9 @@ import type { JsonCompatibleReadOnlyObject } from "../util/index.js";
  * This class refers to the underlying flex tree schema in its constructor, so this class can't be included in the package API.
  */
 export class LeafNodeSchema<Name extends string, const T extends ValueSchema>
-	implements TreeNodeSchemaNonClass<Name, NodeKind.Leaf, TreeValue<T>, TreeValue<T>>
+	implements
+		TreeNodeSchemaNonClass<Name, NodeKind.Leaf, TreeValue<T>, TreeValue<T>>,
+		SimpleLeafNodeSchema
```

Comment:
This interface implementation declaration was missing, so I fixed it. No changes to the type as it already implemented the interface.